desktop/apps/collect/routes.coffee

```diff
@@ -5,15 +5,21 @@ Backbone = require 'backbone'
 JSONPage = require '../../components/json_page'
 page = new JSONPage name: 'browse-categories'
 GeocodedCities = require '../../collections/geocoded_cities'
+{ COLLECT_PAGE_TITLES_URL } = require('sharify').data
+getCollectPageTitle = require '../../components/commercial_filter/page_title'
 
 @index = (req, res, next) ->
   geocodedCities = new GeocodedCities()
+  collectPageTitleFilters = new Backbone.Collection()
   Q.all([
     geocodedCities.fetch()
+    collectPageTitleFilters.fetch(url: COLLECT_PAGE_TITLES_URL)
```

Comment:
is there some default caching on this? the data will not update very frequently... it should be cheap to fetch but if there is a cache available it might be worth using it.

Reply:
Yes, i was just about to add `cache: true` to all 3 of these calls. All of them are static data on S3 (and of course in grand Artsy style: 3 different implementations of fetching from S3!), and so can be cached.

Reply:
if neither filter is set at page load we don't need to fetch the data... it's not 100% clear to me if the promise is lazy evaluated in that case and the url is not fetched.

Reply:
on second thoughts caching is not necessary just for a crawler... just a guarantee of this latter behaviour (lazy fetching).

Reply:
Good call on the optimization to _not_ fetch filters if medium/gene are both not set.

However, then we miss out on the client-side page title updates (which granted, is of minimal benefit, since we won't have loaded that filter <-> title map). Since I'm adding the cache directive (which I think is worth it), we can probably just leave it in all cases. I don't think fetching some JSON from the cache would appreciably add much load time.

Reply:
OK, if you think it's a minimal performance hit - does that `cache:` directive use memcached?

Reply:
Yes, in this case since these are all Backbone models/collections, they use https://github.com/artsy/backbone-super-sync to do the fetching, which supports passing in a cache, which we do here: https://github.com/artsy/force/blob/5199498cfc9f127335b884365548a9dcc4646fac/lib/setup.js#L107

Then, any fetches with `cache: true` go to Force's cache.

Reply:
ok, let's go with that then :+1: